src/core/optimization_container.jl

```diff
@@ -310,8 +310,8 @@ end
 function _make_system_expressions!(
     container::OptimizationContainer,
     bus_numbers::Vector{Int},
-    ::Type{StandardPTDFModel},
-)
+    ::Type{T},
+) where {T <: Union{PTDFPowerModel, StandardPTDFModel}}
```

Comment:
@sourabhdalvi are you sure we need this here? I think that the CopperPlate is handled directly by PM

Reply:
ok, I'll look into that again